rslib/src/notetype/mod.rs

```diff
@@ -330,8 +371,10 @@ impl Notetype {
             return Err(AnkiError::TemplateSaveError(TemplateSaveError {
                 notetype: self.name.clone(),
                 ordinal: idx,
+                details: TemplateSaveErrorDetails::TemplateError,
             }));
         }
+        self.ensure_cloze_if_and_only_if_cloze_notetype(&parsed_templates)?;
```

Comment:
For issues like this that can be checked with a single template, maybe we should just show them in the preview area immediately, instead of waiting until save time to report them?

Reply:
On the other hand, it would mean a bit more processing in the common case of displaying a card. hmm.

Reply:
I think there may also be cases where the template becomes temporarily invalid while the user is working on it, and it would be disruptive to obscure the preview unnecessarily.

Reply:
That happens when the user is typing in something like `{{#Field}}` as well though, and I'm not sure it's such a big problem? :-) But I don't feel strongly either way

Reply:
Let's just keep it the way it is for now then :-)